src/Components/Header.js

```diff
@@ -0,0 +1,10 @@
+const Header = () => {
+   
+    return (
+        <div className="Header">
+           React and Morty go crazy
+        </div>
```

Comment:
semantically a `header` tag or a `headline` tag instead of a `div` would be better.